scripts/exgdas_atmos_analysis_wdqms.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+set -x
+
+# Input GSI diagnostic file containing inputs to wdqms.py
+CNVSTAT="${CDUMP}.t${cyc}z.cnvstat"
+
+# Input files from CNVSTAT fed to wdqms.py
+INPUT_LIST=("diag_conv_ps_ges.${PDY}${cyc}.nc4" \
+            "diag_conv_t_ges.${PDY}${cyc}.nc4" \
+            "diag_conv_q_ges.${PDY}${cyc}.nc4" \
+            "diag_conv_uv_ges.${PDY}${cyc}.nc4")
+
+# Observation types being processed by wdqms.py
+OTYPES=(SYNOP TEMP MARINE)
+
+################################################################################
+echo "Begin job ${job:-}"
+
+#-------------------------------------------------------------------------------
+# Enter working directory
+cd "${DATA}" || ( echo "FATAL ERROR: Unable to cd ${DATA}, ABORT!"; exit 2 )
+
+#-------------------------------------------------------------------------------
+# Copy cnvstat file from COMIN to DATA, untar and gunzip input files for wdqms.py
+cp "${COMIN}/${CNVSTAT}" .
+rc=$?
+(( rc != 0 )) && ( echo "FATAL ERROR: Unable to copy ${CNVSTAT} from ${COMIN}, ABORT!"; exit 2 )
+for file in "${INPUT_LIST[@]}"; do
+  tar -xvf "${CNVSTAT}" "${file}.gz"
+  rc=$?
+  (( rc != 0 )) && ( echo "FATAL ERROR: Unable to extract ${file}.gz from ${CNVSTAT}, ABORT!"; exit 3 )
+  gunzip "${file}.gz"
+  rc=$?
+  (( rc != 0 )) && ( echo "FATAL ERROR: Unable to gunzip ${file}.gz, ABORT!"; exit 3 )
+done
+
+#-------------------------------------------------------------------------------
+# Loop over observation types and produce csv files
+for otype in "${OTYPES[@]}"; do
+  echo "Processing ... ${otype}"
+  ${WDQMSPY} -i ${INPUT_LIST[@]} -t "${otype}" -o "${DATA}"
+  rc=$?
+  if (( rc != 0 )); then
+    echo "FATAL ERROR: wdqms.py failed to process observation type ${otype}; ABORT!"
+    exit "${rc}"
+  fi
+done
+
+#-------------------------------------------------------------------------------
+# Copy output from wdqms.py to COMOUT
+for otype in "${OTYPES[@]}"; do
+  file="NCEP_${otype}_${PDY}_${cyc}.csv"
+  cp "./${file}" "${COMOUT}/${file}" || ( echo "FATAL ERROR: Unable to copy ${file} to ${COMOUT}, ABORT!"; exit 2 )
+done
+
+#-------------------------------------------------------------------------------
+# Send DBN alerts for dataflow to pick up data from COMOUT
+for otype in "${OTYPES[@]}"; do
+  file="NCEP_${otype}_${PDY}_${cyc}.csv"
+  if [[ -f "${COMOUT}/${file}" ]]; then
+    echo "Send DBN Alert"  # TODO: Add appropriate DBNALERT call
+  else
+    echo "WARNING: wdqms.py did not produce '${file}'"  # TODO: Should this be a fatal error?
+  fi
+done
```

Comment:
@aerorahul 
Two questions: 
1. The DBN alerts will trigger dataflow to pick up data from the COMOUT.  Where are these files going (the destination)?
2. Is the destination where our customers can pick up the data they need?   Currently, we put the CVS files at EMCRZDM FTP site for EMCWF to pick up the WDQMS CSV files.

Reply:
I tested three more cycles.  The `gdaswdqms` process ran without problems with the latest `feature/v16.wdqms branch`.

Reply:
I am working on turning those "FATAL ERRORS" into "WARNING". So I might ask for a review again/test.  Apologies for that.

I will leave the DBN Alert for NCO to add.  
I am guessing they will be posted to nomads in a directory `wdqms/` directory under for e.g. `https://nomads.ncep.noaa.gov/pub/data/nccf/com/gfs/prod/gdas.20240311/00/atmos/`
Customers can expect to pick them up from there.

Reply:
> I am working on turning those "FATAL ERRORS" into "WARNING". So I might ask for a review again/test. Apologies for that.
> 
> I will leave the DBN Alert for NCO to add. I am guessing they will be posted to nomads in a directory `wdqms/` directory under for e.g. `https://nomads.ncep.noaa.gov/pub/data/nccf/com/gfs/prod/gdas.20240311/00/atmos/` Customers can expect to pick them up from there.

@aerorahul Thanks for working on adding WDQMS to the globa-workflow and answering my questions.
No problem at all.  I am happy to work on the test.

Reply:
I have turned the `FATAL ERROR` into a `WARNING` if `wdqms.py` fails.
I have also consolidated the loop over the observation types so that when its processing one, it copies it and sends DBNAlert immediately rather than waiting to process all.

Reply:
> I have turned the `FATAL ERROR` into a `WARNING` if `wdqms.py` fails. I have also consolidated the loop over the observation types so that when its processing one, it copies it and sends DBNAlert immediately rather than waiting to process all.

I tested the `gdaswdqms` step again with the update.  It worked well.

Reply:
Thanks @emilyhcliu 
I think this PR is ready for merging. 
If you could open a PR with the details that need to go into the release notes, we can prepare a code delivery and hand over the tag to NCO. 
There is no rush on this, so feel free to prioritize as you see fit.

Reply:
> Thanks @emilyhcliu I think this PR is ready for merging. If you could open a PR with the details that need to go into the release notes, we can prepare a code delivery and hand over the tag to NCO. There is no rush on this, so feel free to prioritize as you see fit.

@aerorahul Thanks for your work.  
I will open a global workflow PR and include detailed information needed by the release note.